name.py

```diff
@@ -0,0 +1,3 @@
+name = input("escribe tu nombre bro:")
+last_name = inputt("Escribe tu apellido bro: ")
```

Comment:
Oye Mau, parece que hay un error en la llamada a la funcion 😢 
```suggestion
last_name = input("Escribe tu apellido bro: ")
```